Disable writing/reading of output file for this test as it seems it may be
called twice in some circumstances (and reading the output modifies the 
results). 

diff --git a/Tests/12.Solvers/quantum-dot-base.fdf b/Tests/12.Solvers/quantum-dot-base.fdf
index b11e59652..8163aa4c5 100644
--- a/Tests/12.Solvers/quantum-dot-base.fdf
+++ b/Tests/12.Solvers/quantum-dot-base.fdf
@@ -14,7 +14,7 @@ use-tree-timer T
 
 WriteDMHS.NetCDF F
 WriteDM.NetCDF F
-WriteDM T
+WriteDM F
 
 SystemName          Si Quantum Dot
 SystemLabel         si-quantum-dot
@@ -42,6 +42,6 @@ MaxSCFIterations    40
 DM.MixingWeight      0.3 
 DM.NumberPulay       4  
 DM.Tolerance         1.d-3
-DM.UseSaveDM             
+DM.UseSaveDM     F
 
 ElectronicTemperature  25 meV  
